testing/sdk_tasks.py

```diff
@@ -116,3 +117,8 @@ def fn():
 
     if exit_status != 0:
         raise RuntimeError('Failed to kill task with pattern "{}", exit status: {}'.format(pattern, exit_status))
+
+
+def task_exec(task_name, command, **kwargs):
```

Comment:
Does this replace the separate implementations in some of the `test_*` files?

Reply:
There was only one instance of that, in `test_soak.py` for the helloworld tests, which was updated.

Reply:
(sorry, I was probably looking at outdated code)